internal/config/config.go

```diff
@@ -1,163 +1,64 @@
 package config
 
 import (
-	"encoding/json"
-	"fmt"
-	"os"
-	"path/filepath"
-
 	"github.com/IBM/sarama"
-	"github.com/kubev2v/migration-planner/internal/util"
-	"sigs.k8s.io/yaml"
+	"github.com/kelseyhightower/envconfig"
 )
 
 const (
-	appName = "migration-planner"
+	appName         = "migration-planner"
+	EnvConfigPrefix = "MIGRATION_PLANNER"
```

Comment:
is this used? You already  prefixed the config with this prefix

Reply:
Yes, you right
This is actually not used
I removed it